client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -121,12 +113,12 @@ class PurchasesListing extends Component {
 			return getJetpackProductTagline( purchase );
 		}
 
-		const jetpackBackupPurchase = this.getJetpackBackupPurchase();
+		const productPurchases = this.getProductPurchases();
 
 		if ( currentPlanSlug ) {
 			const planObject = getPlan( currentPlanSlug );
 			return (
-				planObject.getTagline?.( jetpackBackupPurchase?.productSlug ) ??
+				planObject.getTagline?.( productPurchases[ 0 ]?.productSlug ) ??
```

Comment:
What is the purpose of passing `productSlug` as an argument? I tried with `"jetpack_scan"`, `"jetpack_search"`, and `"jetpack_backup_realtime"` and it always returned `"Your site is being secured and you have access to marketing tools and priority support."`.

Reply:
I think the main idea is to provide a default tagline. 

Do you think we should remove it?

Reply:
In that case, I would leave as it is. We may have, in the future, different strings for each product!